app/routes/forum/components/ThreadList.tsx

```diff
@@ -0,0 +1,40 @@
+import { Flex, LoadingIndicator } from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import { fetchThreads } from 'app/actions/ForumActions';
+import { Content, ContentMain } from 'app/components/Content';
+import { selectThreadsByForumId } from 'app/reducers/threads';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+import ThreadListEntry from './ThreadListEntry';
+import type { ID } from 'app/models';
+import type { PublicThread } from 'app/store/models/Forum';
+
+const ThreadList = ({ forumId }: { forumId: ID }) => {
+  const dispatch = useAppDispatch();
+
+  usePreparedEffect(
+    'fetchAllThreadsByForum',
+    () => dispatch(fetchThreads()),
+    []
+  );
```

Comment:
Shouldn't you just fetch the threads for this specific forum?

Reply:
Ah you're right, I actually don't think I need to fetch them here they should be on the parent forum already.